remote.rb

```diff
@@ -8,19 +8,75 @@
 
 page = Nokogiri::HTML(browser.html)
 
-data = []
+data_arr = []
+
+
+itter = 1
 
 page.css('.owwb-cs-slide-list-account-item').each do |i|
     account  = i.css('.contract-number').text
     balance  = i.css('.owwb-cs-slide-list-amount-value').text
     currency = i.css('.owwb-cs-slide-list-amount-currency').text
 
-data.push(  
-    name:     account,
-    balance:  balance,
-    currency: currency
-)  
-end  
+    pag = []
+    i.css('li').each{|li|  
```

Comment:
for all blocks use [this style guide](https://github.com/bbatsov/ruby-style-guide#spaces-braces)